.github/ISSUE_TEMPLATE.md

```diff
@@ -21,25 +21,7 @@ Please use the following template to assist with creating an issue and to ensure
 
 ### Sample Code or Link to Sample Code
 
-
-### Debug output
-
-Compile clap with `debug` feature:
-
-```toml
-[dependencies]
-clap = { version = "*", features = ["debug"] }
+```rust
+// You code goes here. Please try to write a complete program,
+// including `fn main() { ... }`
 ```
-
-The output may be very long, so feel free to link to a gist or attach a text file
-
-<details>
-<summary> Debug Output </summary>
```

Comment:
Why do you want to remove the debug output sections? It definitely helps us when looking into parser bugs.